src/components/ItemTile/ItemTile.module.scss

```diff
@@ -1,5 +1,6 @@
 @import '../../assets/styles/theme';
 
+@media (max-width: 768px){
```

Comment:
Aby nie powtarzać tak dużo kodu css, to proponuję usunąć to media query, a w tym drugim media query dla desktopu poniżej tylko poustawiać wartości, które się zmieniają względem pierwszego. Bo takie wartości jak `display flex` czy `align-items` pozostają takie same więc nie ma co się powtarzać.